ACadSharp/ACadSharp.csproj

```diff
@@ -14,6 +14,11 @@
     <Description>C# library to read/write cad files like dxf/dwg.</Description>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>ACadSharp.Tests</_Parameter1>
```

Comment:
This comment may be a little picky but to test internals I've added a second Test project `ACadSharpInternal.Tests` which is meant to test any internal in the project that would need it, like partial readers or writers, so in this case I would prefer to remove the project to be allowed to use internals.

Also the project has a [AssemblyInfo](https://github.com/DomCR/ACadSharp/blob/master/ACadSharp/AssemblyInfo.cs) in the root, if you need you can add this information directly there.

Reply:
Reverted this file back to master.  Sounds good to me.